koha-gitify

```diff
@@ -155,6 +156,20 @@ copy_wrapper($aconf, "$aconf.bkp");
 );
 build("$aconf.bkp", $aconf, \%replaces);
 
+# process plack.psgi
+my $plack_file = "/etc/koha/plack.psgi"; # default plack file
+if ( -e $plack_file) {
+    # backup the file
+    copy_wrapper( $plack_file, "$plack_file.bkp" );
+    %replaces = (
+        '/usr/share/koha/lib'               => $gitcheckout,
+        '/usr/share/koha/intranet/cgi-bin'  => $gitcheckout,
+        '/usr/share/koha/api'               => "$gitcheckout/api",
+        '/usr/share/koha/opac/cgi-bin/opac' => "$gitcheckout/opac"
```

Comment:
I generally try to leave a trailing comma after the last item, I can fix in a followup.